aircraft/global5000/Systems/Autopilot.xml

```diff
@@ -0,0 +1,195 @@
+<?xml version="1.0"?>
```

Comment:
Is this version any different to the version in jsbsim/systems?

Reply:
no they are the same